apps/docs/pages/guides/platform/database-usage.mdx

```diff
@@ -23,20 +23,44 @@ This value is also reported in the [database settings page](https://app.supabase
 ## Database storage management
 
 Supabase uses network-attached storage to balance performance with scalability.
-For Pro and Enterprise projects, disk size expands ~1.5x automatically (e.g., 8GB -> 12GB) when you reach 90% of the disk size.
-Disk size expansion can only occur once every six hours.
-Pro projects can store up to 1024TB.
 
-All projects enter read-only mode when you reach 95% of the disk size. In read-only mode, clients will encounter errors such as `cannot execute INSERT in a read-only transaction`.
+For Pro and Enterprise tier projects, [disk size expands ~1.5x automatically](#paid-tier-automatic-disk-resizing) when you reach 90% of the disk size. Disk size expansion can only occur once every six hours.
+
+For Free tier projects, the database will enter ['read only' mode when you reach 95% disk space](#free-tier-project-read-only-mode).
+
+Pro and enterprise tier projects can store up to 1024TB.
+
+### Free tier project 'read-only' mode
+
+Free tier projects enter read-only mode when you reach 95% of the disk size.
+In read-only mode, clients will encounter errors such as `cannot execute INSERT in a read-only transaction`.
 Regular operation (read-write mode) is automatically re-enabled once usage is below 95% of the disk size.
 
+For [automatic disk resizing](paid-tier-automatic-disk-resizing), and avoid your database entering read-only mode, you can [upgrade to the Pro or Enterprise plan](https://app.supabase.com/project/_/settings/billing/subscription).
```

Comment:
Again, I think this is covered in the section below (Increasing available disk size), needn't duplicate. And the below section suggests both alternatives - upgrading to pro or reduce disk usage.

Reply:
Think this is fixed now.

There are 2 distinct scenarios about this though, there is the situation in which someone has a free tier project ending up in read only, and also paid tier. You might need to re-review though.